### PR TITLE
fix(daterange): highlight preset on date change

### LINKED ISSUE #939: Can't pre-select a preset radio button on the date-range  component
When you  add radio buttons for preset ranges on the date range component it doesn't seem to be possible to pre-select one when you bring up the modal.

We are currently replacing our interval picker with the date range component and want to be able to indicate the default range for the dashboard when the dashboard is first displayed. And then as the user chooses different ranges via the buttons I think they would expect them to still be set on subsequent visits to the directive.

### PR DESCRIPTION
**Pull request by andrew-frueh**
check an approproate preset if the currently selected range matches it

closes #939

### CODE REVIEW

File: projects/cashmere/src/lib/date-range/picker-overlay/picker-overlay.component.ts [hunk 1]
```diff
@@ -76,6 +85,19 @@ export class PickerOverlayComponent implements OnInit, AfterViewInit {
         this._setValidity();
     }
 
+    _isRangePreset() {
```
Comment (joeskeen) [inline projects/cashmere/src/lib/date-range/picker-overlay/picker-overlay.component.ts]: @andrew-frueh it sure would be nice to have some unit tests around this... ;)

**Comment by andrew-frueh** [comment]
Lol - you are so right, Joe. I’ll get one added :-)

**Comment by andrew-frueh** [comment]
Unit test added @joeskeen - let me know how that looks.

**Pull request merged**